src/component/feature/FeatureToggleList/FeatureToggleListContainer.tsx

```diff
@@ -1,24 +1,24 @@
 import { useFeatures } from 'hooks/api/getters/useFeatures/useFeatures';
-import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
-import { useFeaturesFilter } from 'hooks/useFeaturesFilter';
-import { FeatureToggleList } from './FeatureToggleList';
-import { useFeaturesSort } from 'hooks/useFeaturesSort';
+import { FeatureSchema } from 'openapi';
+import { FeatureToggleListTable } from './FeatureToggleListTable/FeatureToggleListTable';
+
+const featuresPlaceholder: FeatureSchema[] = Array(7)
+    .fill({
+        name: 'Name of the feature',
+        description: 'Short description of the feature',
+        type: '-',
+        createdAt: new Date(2022, 1, 1),
+        project: 'projectID',
+    })
+    .map((feature, index) => ({ ...feature, name: `${feature.name}${index}` }));
```

Comment:
Could probably remove the `map`, doesn't look like we need unique feature names here.